js_basics/fibonacci_sequence/index.js

```diff
@@ -0,0 +1,15 @@
+const args = process.argv.slice(2);
+
+if (!args[0]) {
+    console.log("A number arguement is required");
+} else if (args[0] < 0) {
+    console.log("An integer number is required.")
+} else {
```

Comment:
Simplify this script by removing the else block.
This can be done by "exiting early" with process.exit(1) if the argument is not a valid number